packages/grid_client/src/modules/models.ts

```diff
@@ -832,6 +832,14 @@ class GetActiveContractsModel {
   @Expose() @IsInt() @IsNotEmpty() @Min(1) nodeId: number;
 }
 
+class USDModel {
```

Comment:
Replace both with CurrencyModel with amount field

Reply:
let's provide a docString for each newly implemented `class` and `methods`, and try to describe what it can do, what parameters are needed, and what it returns